silverstripe/silverstripe-userforms
# Make sticky button not limited to Pages section

## Description
The issue is within the elemental block. The CSS and JS for the sticky button expect the form fields to be displayed within the page's site tree view only. The top part of the fields is overlayed with the buttons bar and in mobile the buttons are visible and if you add many fields there is a big gap at the bottom of the page.

<img width="1164" alt="Screenshot 2024-05-24 at 11 05 46 PM" src="https://github.com/silverstripe/silverstripe-userforms/assets/166450/ca02aec3-1744-4f84-a41d-9912b03e957d">

## Manual testing steps
- Install fresh CMS 
- Add user forms module.
- Add elemental module https://github.com/dnadesign/silverstripe-elemental-userforms
- Create a user form page and add a large number of fields (any type). The form works fine.
- Create a page with an elemental block, add a form block with a large number of fields
- The top part of the fields list is covered 
- scroll to the bottom of the page, and you will see a large empty space
- resize the window to mobile, the buttons are not visible.

## Issues

- https://github.com/dnadesign/silverstripe-elemental-userforms/issues/71

## Pull request checklist

    - See [picking the right version](https://docs.silverstripe.org/en/contributing/code/#picking-the-right-version)
    - Small amounts of additional linting are usually okay, but if it makes it hard to concentrate on the relevant changes, ask for the unrelated changes to be reverted, and submitted as a separate PR.
